src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -87,6 +87,8 @@ interface BuildConfiguration {
 
     fun fetchBuildTags(): List<String>
 
+    fun fetchBuildDependencyTriggers(): List<BuildDependencyTrigger>
```

Comment:
What is 'build dependency trigger'? I see no such thing in TeamCity UI and its [documentation](https://confluence.jetbrains.com/dosearchsite.action?where=TCD10&spaceSearch=true&queryString=build+dependency+trigger). If you mean 'Build Finish Trigger' you should name the class accordingly, otherwise it's hard to understand what it is about.

Let's repeat the general rule again: names in client API should not repeat internal names in responses of REST commands. This library is designed exactly to hide such internal unimportant details from users.